src/blog/2024/08/comparing-dashboard-2-with-uibuilder.md

```diff
@@ -0,0 +1,233 @@
+
+Migrating from Node-RED Dashboard 1 to UI-Builder is significantly more complex. UI-Builder does not follow the same concepts or provide the same UI elements as the Node-RED Dashboard. Users will need to recreate their dashboards from scratch, as UI-Builder relies on custom coding and frontend frameworks rather than the predefined, low-code widgets of Node-RED Dashboard. This process can be overwhelming and requires a solid understanding of HTML, CSS and the frontend frameworks if you wanted use.
+
+## How Active is the Project's Development?
+
+### Node-RED Dashboard 2.0
+
+[Node-RED Dashboard 2.0](https://github.com/FlowFuse/node-red-dashboard/graphs/contributors), which replaced Node-RED Dashboard 1.0 in 2023, has shown consistent and high development activity. The project benefits from a dedicated team that regularly updates and improves it, ensuring it remains current with user needs and technological advancements.
+
+![Screenshot of the Node-RED Dashboard 2.0 GitHub commit chart](./images/dashboard-2-commits.png)  
+_Screenshot of the Node-RED Dashboard 2.0 GitHub commit chart._
+
+### UI-Builder
+
+[UI-Builder](https://github.com/TotallyInformation/node-red-contrib-uibuilder/graphs/contributors) has been an active project for a long time and remains active. However, there has been a noticeable decline in development activity starting in early 2024. While the project continues to be maintained, it does not have a dedicated team and does not receive regular updates as frequently as some other projects.
+
+![Screenshot of the UI-Builder GitHub commit chart](./images/ui-builder-commits.png)  
+_Screenshot of the UI-Builder GitHub commit chart._
+
+## How Extensive is the Collection of UI Elements?
+
+### Node-RED Dashboard 2.0
+
+Node-RED Dashboard 2.0 offers an extensive set of UI elements, including forms, dropdowns, tables, charts, and gauges that are super easy to use. These widgets are built with complex Vue.js components, but users are completely shielded from this complexity, allowing them to focus on ease of use. Additionally, if you want to create your own custom widget, you can do so with the ui-template node that accepts complete Vue components. By default, Dashboard 2.0 supports the Vuetify library, which provides an extensive set of UI components.
+
+## UI-Builder
+
+UI-Builder also offers a number of widgets, but these are not as user-friendly as those in Dashboard 2.0. Users must send a JSON config object, which can be complex for new users who lack knowledge of HTML/CSS. Additionally, handling widget data requires using UI-Builder's methods, which can further increase complexity. However, UI-Builder's strength lies in its flexibility, allowing any HTML element to be used as a component, and it also supports the W3C standard web components. Despite this, users need to perform a lot of additional tasks to get everything set up and will have the hassle of writing things.
+
+## How much dynamic insertion or update it support?
```

Comment:
Not sure this title is clear. I know what you're trying to say, as I built dynamic options into Dashboard - but think about the user value here.

Something like "Controlling UI at runtime"?